README.md

```diff
@@ -72,7 +72,7 @@ chart-test.sh
 ### Linting Charts
 
 ```shell
-docker run --rm -v "$(pwd):/workdir" --workdir /workdir gcr.io/kubernetes-charts-ci/chart-testing:v1.0.2 chart_test.sh --no-install --config .mytestenv
+docker run --rm -v "$(pwd):/workdir" --workdir /workdir gcr.io/kubernetes-charts-ci/chart-testing:v1.0.3 chart_test.sh --no-install --config .mytestenv
```

Comment:
I feel like I should know this, but what's our process for tagging/pushing this image?

Reply:
There's not really a process. So far I've been pushing the images manually which is not ideal. See #4.